diff --git a/index.js b/index.js
index a9e3f98405c10bd34db4c4a192c5b490fad3bc31..5dcf91927301ce9a146aafc41775fa936b8ad188 100644
--- a/index.js
+++ b/index.js
@@ -58,7 +58,7 @@ export default function (options = {}) {
 				loader: {
 					'.wasm': 'copy'
 				},
-				external: ['cloudflare:*']
+				external: ['cloudflare:*', 'node:*', 'stream', 'string_decoder', 'util', 'path', 'tls', 'fs', 'net', 'crypto', 'dns', 'assert'],
 			});
 		}
 	};
